File: src/test/java/com/flowcommerce/core/domain/member/MemberFixture.java
package com.flowcommerce.core.domain.member;

public class MemberFixture {
    public static SignUp createSignUp() {
        return new SignUp("test@test.com", "Password1!", "test");
    }

    public static SignUp createSignUp(String email, String password, String name) {
        return new SignUp(email, password, name);
    }

    public static SignUp createSignUpWithEmail(String email) {
        return new SignUp(email, "Password1!", "test");
    }
}

File: src/test/kotlin/com/flowcommerce/core/domain/member/MemberFixture.kt
package com.flowcommerce.core.domain.member

object MemberFixture {
    fun createSignUp(): SignUp = SignUp("test@test.com", "Password1!", "test")

    fun createSignUp(email: String, password: String, name: String): SignUp =
        SignUp(email, password, name)

    fun createSignUpWithEmail(email: String): SignUp =
        SignUp(email, "Password1!", "test")
}
